src/styles/variables.scss

```diff
@@ -0,0 +1,7 @@
+// Colors
+
+$main-accent: #060B35;
+$secondary: #616070;
+$blue-accent: #00ACDC;
+$gray-elements: #F3F3F3;
+$white: #FFF;
```

Comment:
- Remove comments
- Add a blank line at the end of the file, fix it in all places
- Variable names should describe their purpose. If the designer changes the color to red instead, it will be strange when `$color-blue` will be red. For example `$main-color`